code/game/objects/items/sticker.dm

```diff
@@ -0,0 +1,138 @@
+/obj/item/sticker
+	name = "sticker"
+	desc = "A sticker with some strong adhesive on the back, sticks to stuff!"
+	item_flags = NOBLUDGEON | XENOMORPH_HOLDABLE //funny
+	resistance_flags = FLAMMABLE
+	icon = 'icons/obj/stickers.dmi'
+	w_class = WEIGHT_CLASS_TINY
+	throw_range = 3
+	vis_flags = VIS_INHERIT_DIR | VIS_INHERIT_PLANE | VIS_INHERIT_LAYER
+	///The overlay we apply to things we stick to
+	var/mutable_appearance/sticker_overlay
+	///A list of icon_states to pick an icon_state on Initialize, provided it is not null.
+	var/list/icon_states
+	///The thing we are attached to
+	var/atom/attached
+	///The turf our COMSIG_TURF_EXPOSE is registered to, so we can unregister it later.
+	var/turf/signal_turf
+
+/obj/item/sticker/Initialize(mapload)
+	. = ..()
+	if(icon_states)
+		icon_state = pick(icon_states)
+	pixel_y = rand(-3,3)
+	pixel_x = rand(-3,3)
+
+/obj/item/sticker/afterattack(atom/target, mob/living/user, prox, params)
+	. = ..()
+	if(!prox)
+		return
+	if(!isliving(target) && !isobj(target) && !isturf(target))
+		return
+	user.visible_message(span_notice("[user] sticks [src] to [target]!"),span_notice("You stick [src] to [target]!"))
```

Comment:
The visual feedback of the sticker is enough.

I would like for this to play the attacking animation instead of any to_chat so it's obvious who is doing it, and it would look a lot better anyway.